### PR TITLE
Fix batch labels

### PR DESCRIPTION
**Pull request by Beacze**

### CODE REVIEW

File: Kitodo/src/main/java/org/kitodo/production/services/data/BatchService.java [hunk 1]
```diff
@@ -303,39 +305,45 @@ public int size(Batch batch) {
      * batch.
      *
      */
-    public String toString(Batch batch) {
+    public void toString(Batch batch) {
```
Comment (Kathrin-Huber) [inline Kitodo/src/main/java/org/kitodo/production/services/data/BatchService.java]: shouldn't this be named "createLabel" or something?

File: Kitodo/src/main/java/org/kitodo/production/helper/tasks/ExportSerialBatchTask.java [hunk 2]
```diff
@@ -80,7 +80,7 @@
      *            batch holding a serial publication
      */
     public ExportSerialBatchTask(Batch batch) {
-        super(batch.getLabel());
+        super(ServiceManager.getBatchService().getLabel(batch));
```
Comment (Kathrin-Huber) [inline Kitodo/src/main/java/org/kitodo/production/helper/tasks/ExportSerialBatchTask.java]: getLabel is not using the label you created

File: Kitodo/src/main/java/org/kitodo/production/forms/BatchForm.java [hunk 3]
```diff
@@ -161,6 +164,9 @@ public void filterBatches() {
                 currentBatches.add(batch);
             }
         }
+        for (Batch batch : this.currentBatches) {
```
Comment (Kathrin-Huber) [inline Kitodo/src/main/java/org/kitodo/production/forms/BatchForm.java]: can't you create the label in "getLabel" if there is no label?
I think it's not intuitive to create the label beforehand

File: Kitodo/src/main/java/org/kitodo/production/helper/tasks/ExportSerialBatchTask.java (hunk 2 above)
Comment (Beacze) [reply Kitodo/src/main/java/org/kitodo/production/helper/tasks/ExportSerialBatchTask.java]: Yeah, it is different label. The one which overwrites toString is used only for frontend.

File: Kitodo/src/main/java/org/kitodo/production/forms/BatchForm.java (hunk 3 above)
Comment (Beacze) [reply Kitodo/src/main/java/org/kitodo/production/forms/BatchForm.java]: Yeah, but it is different label. I don't know why, but this class creates different label in getLabel and different in toString.

File: Kitodo/src/main/java/org/kitodo/production/services/data/BatchService.java (hunk 1 above)
Comment (Beacze) [reply Kitodo/src/main/java/org/kitodo/production/services/data/BatchService.java]: renamed

**Pull request merged**